app/assets/javascripts/client/directives/formExtensions.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+    .directive('float', [function () {
```

Comment:
Style:  the directive (and link function) should be broken out into their own, separate functions.